mongodb-storage/src/main/java/io/lumeer/mongodb/MongoDbStorageDialect.java

```diff
@@ -65,11 +66,18 @@ public DataDocument addRecentlyUsedDocumentQuery(final String metadataCollection
             .append("update", new DataDocument()
                   .append("$push", new DataDocument()
                         .append(LumeerConst.Collection.RECENTLY_USED_DOCUMENTS_KEY, new DataDocument()
-                              .append("$each", Arrays.asList(id))
+                              .append("$each", Collections.singletonList(id))
                               .append("$position", 0)
                               .append("$slice", listSize))));
    }
 
+   public DataDocument[] usersOfGroupAggregate(final String organization, final String group) {
```

Comment:
I'm afraid that it's hard to make generic method for situations like this. Aggregate phases (and als their count) can be different in each situation. Of course there is a way how to do it more generic, but the method will take lot of params and inside the method we will need loot of conditions...